frontend/app/controllers/Info.scala

```diff
@@ -66,7 +67,17 @@ trait Info extends Controller {
     )
   }
 
-  def supporterAustralia = CachedAndOutageProtected { implicit request =>
+  def supporterAustralia = NoCacheAction { implicit request =>
```

Comment:
In this case it would add quite a bit of complexity - have had a look at the traffic with @rtyley and it looks like it would be fine to decache (in the past half-an-hour both the Australian and US supporter pages have each had between 0 and 3 pageviews per minute)